community/kernel/src/main/java/org/neo4j/kernel/configuration/ssl/SslSystemSettings.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2002-2017 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.kernel.configuration.ssl;
+
+import io.netty.handler.ssl.SslProvider;
+
+import org.neo4j.configuration.Description;
+import org.neo4j.configuration.LoadableConfig;
+import org.neo4j.graphdb.config.Setting;
+
+import static org.neo4j.kernel.configuration.Settings.options;
+import static org.neo4j.kernel.configuration.Settings.setting;
+
+/**
+ * System-wide settings for SSL.
+ */
+@Description( "System-wide settings for SSL." )
+public class SslSystemSettings implements LoadableConfig
```

Comment:
Maybe simply `SslSettings`? Otherwise "ssl system" is slightly confusing.

Reply:
Well, it is to distinguish it from other ssl settings. Simply SslSettings is not good enough.